client/components/notice/notice-action.jsx

```diff
@@ -8,6 +8,11 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Gridicon from 'gridicons';
 
+/**
+ * Style dependencies
+ */
+import './style.scss';
```

Comment:
It's unfortunate that we include this stylesheet twice. I assume you did it because we import `NoticeAction` independently?
```js
import NoticeAction from 'components/notice/notice-action';
```

We can make one cleanup here: the `NoticeAction` is not useful outside `Notice`: we always use both components together:
```jsx
<Notice><NoticeAction /></Notice>
```

A more ergonomic way to import them would be from one module:
```js
import Notice, { NoticeAction } from 'components/notice';
```

This would be a really nice refactor in a followup PR.

Reply:
Yes, I added that because I saw `NoticeAction` has been imported directly. Will add a follow up PR with a refactor after this one. :)